### PR TITLE
Implement #358 (Redirect to comment after moderation)

### LINKED ISSUE #358: Make activate comment link take you directly to the comment
Hi, what do you think about this:

When clicking activate or delete, there would be no confirmation, but it would take you directly to the right comment, so you can reply.

Currently I have to go multiple times between the e-mail and the browser:
1. receive e-mail
1. click activate
1. click activate on the confirmation dialog
1. go back to the e-mail
1. click on the comment link
1. reply

It would be nicer like this:
1. receive e-mail
1. click activate
1. reply

### PR DESCRIPTION
**Pull request by Kwpolska**
This closes #358. After a comment is successfully moderated, the user is redirected to the comment page itself.

### CODE REVIEW

File: isso/views/comments.py [hunk 1]
```diff
@@ -516,14 +517,13 @@ def delete(self, environ, request, id, key=None):
...
         &lt;html&gt;
-            &lt;head&gt;
-                &lt;script&gt;
-                    if (confirm('Delete: Are you sure?')) {
-                        xhr = new XMLHttpRequest;
-                        xhr.open('POST', window.location.href);
-                        xhr.send(null);
-                    }
-                &lt;/script&gt;
+        &lt;head&gt;
```
Comment (jelmer) [inline isso/views/comments.py]: This change appears unrelated, why is it necessary?

Comment (Kwpolska) [reply isso/views/comments.py]: I updated some docs on occasion.

Comment (jelmer) [reply isso/views/comments.py]: Please split that change out to a separate pull request.

Comment (Kwpolska) [reply isso/views/comments.py]: Undone. Feel free to reintroduce yourself, I’m not going to create a PR for this.

Comment (jelmer) [reply isso/views/comments.py]: Thanks; it's just that random unrelated changes make the pull request and subsequent merge commit harder to understand, especially for somebody not intimately familiar with all parts of isso's codebase, like myself.

**Comment by Kwpolska** [comment]
Thanks for merging!
